tests/s3/test_bucket_tagging.py

```diff
@@ -829,3 +830,57 @@ def test_2453(self):
             assert_utils.assert_in(errmsg.NO_BUCKET_OBJ_ERR_KEY, str(error.message), error.message)
         self.log.info("Step 2: Deleting tag of a non existing bucket failed with NoSuchBucket")
         self.log.info("ENDED: Verify DELETE bucket tagging to non-existing bucket")
+
+    @pytest.mark.parallel
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_bucket_tags
+    @pytest.mark.tags("TEST-XXXXX")
+    def test_xxxx(self):
+        """Create bucket tags with encoded k:v pair with base64 encoding."""
+        self.log.info("STARTED: Create bucket tags with encoded k:v with base64 encoding")
+        self.log.info("Step 1: Creating a bucket %s", self.bucket_name)
+        resp = self.s3_obj.create_bucket(self.bucket_name)
+        self.log.info(resp)
```

Comment:
I can see create bucket respons was aleady logged in lib so avoid relogging in test module for such instanece.